packages/name-controller/src/NameController.test.ts

```diff
@@ -513,6 +517,242 @@ describe('NameController', () => {
   });
 
   describe('updateProposedNames', () => {
+    it('does remove entries if all proposed names are expired', async () => {
```

Comment:
Should we put these new tests in a nested `describe` like `removes entries` just to group them a little?

Also is it worth moving these tests to the bottom of the main describe so the core functionality is tested first?

Reply:
Scoped and relocated tests in https://github.com/MetaMask/core/pull/3748/commits/543947f748d2d1814230fc96aa76d97832a23b5f